tests/core/quantms/psms/test_duckdb_creation.py

```
@@ -0,0 +1,670 @@
+        TEST_DATA_ROOT
+        / "quantms/dda-lfq-full/PXD007683-LFQ.sdrf_openms_design_msstats_in.csv.gz"
+    )
+    sdrf_path = TEST_DATA_ROOT / "quantms/dda-lfq-full/PXD007683-LFQ.sdrf.tsv"
+
+    if not mztab_path.exists() or not msstats_path.exists():
+        pytest.skip("Test files not found")
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        output_path = Path(temp_dir) / "test_two_step.duckdb"
+
+        print(f"\n=== Testing Two-Step Creation Process ===")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)